static/js/src/entry/account/constants.js

```diff
@@ -1,23 +1,18 @@
+/* eslint-disable prefer-destructuring */
```

Comment:
Need to disable in this file to ensure we can can assign values to SENTRY_DSN and SENTRY_ENVIRONMENT without raising lint errors.